.circleci/ci-oldest-reqs.txt

```diff
@@ -1,5 +1,10 @@
+click==8.0.4
 cloudpickle==1.1.1
+coverage==6.3.2
```

Comment:
```suggestion
coverage==6.2
```

Reply:
I did not check @bdice's commit and changed this. I will undo this change now.

Reply:
No worries. The oldest (Python 3.6) configuration requires coverage 6.2 for support. All other configurations can be updated.